local-cli/generator-macos/templates/macos/Podfile

```diff
@@ -0,0 +1,40 @@
+# require_relative '../node_modules/@react-native-community/cli-platform-ios/native_modules'
+
+abstract_target 'Shared' do
+  # use_native_modules!
+
+  pod 'React', :path => "../node_modules/react-native-macos/"
+  pod 'React-Core', :path => "../node_modules/react-native-macos/React"
+  pod 'React-fishhook', :path => "../node_modules/react-native-macos/Libraries/fishhook"
+  pod 'React-RCTActionSheet', :path => "../node_modules/react-native-macos/Libraries/ActionSheetIOS"
+  pod 'React-RCTAnimation', :path => "../node_modules/react-native-macos/Libraries/NativeAnimation"
+  pod 'React-RCTBlob', :path => "../node_modules/react-native-macos/Libraries/Blob"
+  pod 'React-RCTImage', :path => "../node_modules/react-native-macos/Libraries/Image"
+  pod 'React-RCTLinking', :path => "../node_modules/react-native-macos/Libraries/LinkingIOS"
+  pod 'React-RCTNetwork', :path => "../node_modules/react-native-macos/Libraries/Network"
+  pod 'React-RCTSettings', :path => "../node_modules/react-native-macos/Libraries/Settings"
+  pod 'React-RCTText', :path => "../node_modules/react-native-macos/Libraries/Text"
+  pod 'React-RCTVibration', :path => "../node_modules/react-native-macos/Libraries/Vibration"
+  pod 'React-RCTWebSocket', :path => "../node_modules/react-native-macos/Libraries/WebSocket"
+  pod 'React-cxxreact', :path => "../node_modules/react-native-macos/ReactCommon/cxxreact"
+  pod 'React-jscallinvoker', :path => "../node_modules/react-native-macos/ReactCommon/jscallinvoker"
+  pod 'React-jsi', :path => "../node_modules/react-native-macos/ReactCommon/jsi"
+  pod 'React-jsiexecutor', :path => "../node_modules/react-native-macos/ReactCommon/jsiexecutor"
+  pod 'React-jsinspector', :path => "../node_modules/react-native-macos/ReactCommon/jsinspector"
+  pod 'yoga', :path => "../node_modules/react-native-macos/ReactCommon/yoga"
+  pod 'DoubleConversion', :podspec => "../node_modules/react-native-macos/third-party-podspecs/DoubleConversion.podspec"
+  pod 'glog', :podspec => "../node_modules/react-native-macos/third-party-podspecs/glog.podspec"
+  pod 'Folly', :podspec => "../node_modules/react-native-macos/third-party-podspecs/Folly.podspec"
+  pod 'boost-for-react-native', :podspec => "../node_modules/react-native-macos/third-party-podspecs/boost-for-react-native.podspec"
+  pod 'React-DevSupport', :path => "../node_modules/react-native-macos/React"
+
+  target 'HelloWorld-macOS' do
+    platform :macos, '10.14'
+    # Pods specifically for macOS target
+  end
+
+  target 'HelloWorld-iOS' do
+    platform :ios, '9'
```

Comment:
We try and do currentVersion-1 for iOS, so probably worth making this 12 unless that'd break with RN still being on 9 somehow.

Also why is there an iOS target in the macos directory/Podfile?

Reply:
> We try and do currentVersion-1 for iOS, so probably worth making this 12 unless that'd break with RN still being on 9 somehow.

Nothing would break, just trying to keep this in sync with upstream.

> Also why is there an iOS target in the macos directory/Podfile?

Great question and one I asked myself too! 😄 

@tom-un Explained to me that he likes to have one in the `macos` dir for now as it can target the `iOS` bits of _our_ RN fork to quickly test if we’ve broken code as opposed to the code in the `ios` dir that targets upstream RN. We did briefly [discuss](https://teams.microsoft.com/l/message/19:55ba60b13640496f96902584ce675b97@thread.skype/1585851262921?tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47&groupId=8034415f-bc0a-4b5d-9453-78ee27e16b22&parentMessageId=1584970392446&teamName=React%20Native%20%40Microsoft&channelName=React%20Native%20macOS&createdTime=1585851262921) to perhaps hide this extra iOS target by default in the future, though.

Reply:
Ya, and I think it'll be easier to implement a `--add-ios-target` flag once this PR lands.   Because it could be implemented as Ruby script.   Right now I think we'd have to have *two* `project.pbxproj` templates: one with only a mac target and another with mac and ios targets and that would be a pain to maintain.